pkgs/by-name/li/libnotify/package.nix

```diff
@@ -48,7 +48,7 @@ stdenv.mkDerivation rec {
       ninja
       pkg-config
       libxslt
-      docbook-xsl-ns
+      docbook_xsl
```

Comment:
This is an alias, the canonical name is `docbook-xsl-nons`. But it is really [incorrect and breaks the man page](https://gitlab.gnome.org/GNOME/libnotify/-/merge_requests/37#note_2366448).

Reply:
Reverted just this match and switched back to `xsl-ns`.